scala-cli

```diff
@@ -17,7 +17,7 @@ if [ ! -x "$LAUNCHER_DIR/scala" ]; then
   echo "native-image launcher not built yet." 1>&2
   echo "In order to build it, go in $DIR, and run" 1>&2
   echo 1>&2
-  echo "  ./mill cli.nativeImage" 1>&2
+  echo "  ./mill copyTo cli.nativeImage out/cli/nativeImage/dest/scala" 1>&2
```

Comment:
Just `./mill cli.nativeImage` should be enough. We should change `LAUNCHER_DIR` if the native image doesn't land there.